lib/components/fullpage.js

```diff
@@ -6,10 +6,20 @@ const SideNav = require('./sideNav');
 const scrollTo = require('../utils/scrollTo');
 const events = require('../utils/events');
 const renderUtils = require('../utils/renderUtils');
-const BROWSER = renderUtils.browser();
-const ELEMENT_BROWSERS = renderUtils.elementBrowsers;
 
-const BODY = !!~ELEMENT_BROWSERS.indexOf(BROWSER) ? document.documentElement : document.body;
+let BROWSER = null;
+let ELEMENT_BROWSERS = null;
+let BODY = null;
+
+const getBody = () => {
+  if (!BODY) {
```

Comment:
will `BODY` ever be pre defined? If you're overriding in react-universal I would think we need to provide an entry point for users that don't use it